lab-kaylyn/app/component/navbar/_navbar.scss

```diff
@@ -0,0 +1,42 @@
+form {
+  text-align: center;
+
+  .form-control {
+    height: 3em;
+  }
+}
+
+.navbar {
+  .btn-logout {
+    display: none;
+    @media (min-width: 500px){
```

Comment:
@kaylynyuh good use of media queries - you may want to look into iOS device specific breakpoints, just as general tool for making major modifications.  Breakpoints should really be tweak points, as you have done here, but iOS devices still dominate the market and we usually still design major layout changes based on standard iOS device widths (ipad, iphone, 13/15" screens)

https://css-tricks.com/snippets/css/media-queries-for-standard-devices/